joomla/joomla-cms
# [4.0] Pagination seo friendly

Pull Request for Issue #26881 

### Summary of Changes
- Appends a `limitstart` param to the action on pagination links


### Testing Instructions
- Ensure the pagination doesn't break with this PR.
- Verify if `limitstart` param is appended & modified while paginating.
- Check if proper page is rendered when `limitstart` is modified in address bar.


### Expected result



### Actual result



### Documentation Changes Required
NIL
